crypto/ec/ec_asn1.c

```diff
@@ -74,6 +74,12 @@ struct ec_parameters_st {
     ASN1_INTEGER *cofactor;
 } /* ECPARAMETERS */ ;
 
+typedef enum {
+    ECPKPARAMETERS_TYPE_NAMED = 0,
+    ECPKPARAMETERS_TYPE_EXPLICIT,
+    ECPKPARAMETERS_TYPE_IMPLICITLYCA
```

Comment:
How about simply `ECPKPARAMETERS_TYPE_IMPLICIT`?

Reply:
Good that you introduced these symbolic names.

Reply:
Renamed